locales/templates/app.pot

```diff
@@ -1132,11 +1132,48 @@ msgctxt "preference_autocomplete_duplicate_url_error"
 msgid "URL already exists"
 msgstr ""
 
-#. Find in page
+#. Label for the Find in page button
 msgctxt "find_in_page"
 msgid "Find in Page"
 msgstr ""
 
+#. Watermark/Hint for the find in page input field
+msgctxt "find_in_page_input"
+msgid "Find in page"
+msgstr ""
+
+#. String to show the number of results found in the page and the
+#. position the user is at. The first argument is the position, the second
+#. argument is the total
+#, c-format
+msgctxt "find_in_page_result"
+msgid "%1$d/%2$d"
+msgstr ""
+
+#. String to be read by the accessibility service
+#, c-format
+msgctxt "accessibility_find_in_page_result"
+msgid "%1$d out of %2$d"
+msgstr ""
+
+#. String to be read by the accessibility service when focusing the next
+#. result button
+msgctxt "accessibility_next_result"
+msgid "Find next result"
+msgstr ""
+
+#. String to be read by the accessibility service when focusing the previous
+#. result button
+msgctxt "accessibility_previous_result"
+msgid "Find previous result"
+msgstr ""
+
+#. String to be read by the accessibility service when focusing the close find
+#. in page button
```

Comment:
Nit: I'd add quotes around "Close find in page" here. The comment is confusing to read otherwise.

Reply:
I'm trying to find a better way so that localizers understand this correctly and make sure it's read on the screenreader in a coherent way in all locales. But I'm still struggling to find that